Document FIFO queue creation (#193)
localstack/localstack-docs: src/content/docs/aws/services/sqs.mdx

    --attribute-names All
```

> To create a FIFO queue with suffix `.fifo`, as used by [AWS FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-fifo-queue-message-identifiers.html), the queue needs to have the attributes `FifoQueue=true` set:

Reviewer: The `.fifo` suffix is a requirement when creating a FIFO queue in addiiton to the `FifoQueue` attribute being set to `true`. Perhaps this should be reworded:

```suggestion
To create a [FIFO queue](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-fifo-queue-message-identifiers.html), the queue name must end with the `.fifo` suffix in addition to the `FifoQueue=true` attribute set:
```

```bash

> awslocal sqs create-queue --queue-name localstack-queue.fifo --attributes "FifoQueue=true"

Reviewer: nit: Should we be doing this in the `Create a queue` section?
